apps/homescreen/test/unit/grid_test.js

```diff
@@ -88,12 +101,15 @@ suite('grid.js >', function() {
     };
     realMozSettings = navigator.mozSettings;
     navigator.mozSettings = MockNavigatorSettings;
-
+    navigator.mozL10n = MockL10n;
+    GridManager._ = MockL10n.get;
```

Comment:
`_` is not exposed on `GridManager` so this line doesn't really help.  It's a private variable in the `GridManager`'s IIFE.